components/molecules/selector/Selector.tsx

```
@@ -89,38 +92,62 @@ export const Selector = <T extends Option>({
         ) : (
           <button
             onClick={handleToggle}
-            className="flex items-center justify-between w-full text-left"
+            className={cn(
+              'flex items-center justify-between w-full py-1 pl-2 text-left'
+            )}
+            aria-haspopup="listbox"
+            aria-expanded={isOpen}
           >
-            <span className="text-text-primary truncate">
+            <span className="h-7 leading-7 text-center text-text-primary truncate flex-1 min-w-0">
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)